File: 02. C++/12. Condicional Switch/Condicional Switch.cpp
#include "iostream"
#include "string"
#include "stdlib.h"

using namespace std;

int main()
{

    int valor = 10;
    
    switch (valor) {
        case 0:
            cout << "El valor es 0" << endl;
            break;
        case 1:
            cout << "El valor es 1" << endl;
            break;
        case 2:
            cout << "El valor es 2" << endl;
            break;
        case 3:
            cout << "El valor es 3" << endl;
            break;
        default:
            cout << "El valor es otro numero" << endl;
    }
        
    system("pause");
    return EXIT_SUCCESS;
    
}

File: 05. Lenguajes no clasificados/01. C#/12. Condicional Switch/Condicional Switch.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace prueba
{
    class Program
    {

        static void Main(string[] args)
        {

            int valor = 10;
            switch (valor) {
                case 0:
                    Console.WriteLine("El valor es 0");
                    break;
                case 1:
                    Console.WriteLine("El valor es 1");
                    break;
                case 2:
                    Console.WriteLine("El valor es 2");
                    break;
                case 3:
                    Console.WriteLine("El valor es 3");
                    break;
                default:
                    Console.WriteLine("El valor es otro numero");
                    break;
            }

            Console.ReadLine();

        }
    }
}
